src/core/net/nat64_translator.hpp

```diff
@@ -379,6 +381,8 @@ class Translator : public InstanceLocator, private NonCopyable
 
     ProtocolCounters mCounters;
     ErrorCounters    mErrorCounters;
+
+    State mState;
```

Comment:
We can consider moving `mState` after `bool mEnabled` so to use the padded bytes?

Reply:
Done